app/models/course.rb

```diff
@@ -2,6 +2,7 @@ class Course < ApplicationRecord
   include Discard::Model
   include WithQualifications
   include ChangedAt
+  include TouchProvider
```

Comment:
@stellenberger - looks like we may overlapping cards! Was about to raise a PR for the same change
https://trello.com/c/RsbjA4sq/3299-incrementally-sync-courses-from-the-ttapi

My only suggestion here is rather than using the TouchProvider concern you can simply do
```
belongs_to :provider, touch: :changed_at
```

... this means the TouchProvider and TouchCourse concerns could be deleted in another PR but I'll let @gpeng @defong  and company make a call on this

Reply:
I prefer @TobyRet's suggestion. I'm not a fan of the `changed_at` field either but if we already have everything built around that rather than `updated_at` then we'll have to press on with that for now.

Reply:
The `TouchProvider` module will only change the `changed_at` field for the provider when we touch it, and not the `updated_at` as it uses `update_columns`. I am happy to implement @TobyRet's suggestion, though it will end up changing both `updated_at` and `changed_at` fields for the provider when we touch it. If it's not important that both fields are updated, I'll push up @TobyRet's suggestion! 👍